app/components/blacklight/advanced_search_form_component.rb

```diff
@@ -31,7 +31,7 @@ def sort_fields_select
       options = sort_fields.values.map { |field_config| [helpers.sort_field_label(field_config.key), field_config.key] }
       return unless options.any?
 
-      select_tag(:sort, options_for_select(options, params[:sort]), class: "form-control sort-select w-auto")
+      select_tag(:sort, options_for_select(options, params[:sort]), class: "form-select sort-select w-auto")
```

Comment:
```suggestion
      select_tag(:sort, options_for_select(options, params[:sort]), class: "form-select custom-select sort-select w-auto")
```

Do we need to add `custom-select` or keep `form-control` for Bootstrap 4-compatible styling?